src/components/graph/util/model-constants.js

```diff
@@ -25,6 +25,7 @@ export const EQUIPMENT_TYPE_LABEL_KEYS = {
     [EQUIPMENT_TYPES.LINE]: 'Line',
     [EQUIPMENT_TYPES.TWO_WINDINGS_TRANSFORMER]: '2WTransformer',
     [EQUIPMENT_TYPES.THREE_WINDINGS_TRANSFORMER]: '3WTransformer',
+    [EQUIPMENT_TYPES.HVDC_LINE]: 'HvdcLine',
```

Comment:
```suggestion
    [EQUIPMENT_TYPES.HVDC_LINE]: 'HVDCLine',
```

Reply:
Why is this one resolved ? Did you discuss about it ?

Reply:
It is not used for translation but rather for generating combined translation keys with a prefix/suffix for each equipment type.